### PR TITLE
samples: wifi: Add build support for nrf7001 DK board

### PR DESCRIPTION
**Pull request by D-Triveni**
Add change to build scan, sta and radio_test samples for nrf7001 dk board.

Fixes SHEL-1627.

### CODE REVIEW

**Review by sachinthegreen — APPROVED** [review]

**Review by krish2718 — CHANGES_REQUESTED** [review]
Not adding 7001 to tests was intentional to minimize unnecessary twister tests, see https://github.com/nrfconnect/sdk-nrf/pull/10964#discussion_r1187179501

File: samples/wifi/sta/Kconfig [hunk 1]
```diff
@@ -49,7 +49,7 @@ config STA_SAMPLE_PASSWORD
 
 config NRF700X_QSPI_ENCRYPTION_KEY
 	string "16 bytes QSPI encryption key, only for testing purposes"
-	depends on BOARD_NRF7002DK_NRF5340_CPUAPP
+	depends on (BOARD_NRF7002DK_NRF5340_CPUAPP || BOARD_NRF7002DK_NRF7001_NRF5340_CPUAPP)
```
[the remark this answers is not in the document]
Comment (D-Triveni) [reply samples/wifi/sta/Kconfig:52]: Will address and raise a new PR for this change.

**Comment by D-Triveni** [comment]
> Not adding 7001 to tests was intentional to minimize unnecessary twister tests, see [#10964 (comment)](https://github.com/nrfconnect/sdk-nrf/pull/10964#discussion_r1187179501)

So, closing this PR.

File: samples/wifi/sta/Kconfig (hunk 1 above)
[the remark this answers is not in the document]
Comment (krish2718) [reply samples/wifi/sta/Kconfig:52]: You can just push to this PR, it will have a context.

**Pull request closed without merging**